### PR TITLE
fix: remove - from cert-manager v1upgrade.yaml

### PR DESCRIPTION
**Pull request by armandgrillet**
**What type of PR is this?**
Bug.

**What this PR does/ why we need it**:
This was causing issues when upgrading the `cert-manager` addon.

**Which issue(s) this PR fixes**:
No issue.

**Special notes for your reviewer**:
Related docs: https://helm.sh/docs/chart_template_guide/control_structures/#controlling-whitespace (thanks @makkes)

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Checklist**

* [X] *If a chart is changed, the chart version is correctly incremented.*
* [X] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: stable/cert-manager-setup/templates/v1upgrade.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{- if .Values.installCertManager }}
-{{- if (.Capabilities.APIVersions.Has "certmanager.k8s.io/v1alpha1") -}}
+{{- if (.Capabilities.APIVersions.Has "certmanager.k8s.io/v1alpha1") }}
 --
```
Comment (makkes) [inline stable/cert-manager-setup/templates/v1upgrade.yaml:3]: I think the actual bug is here:
```suggestion
---
```

**Review by hectorj2f — APPROVED** [review]

**Review by makkes — APPROVED** [review]

**Review by jimmidyson — APPROVED** [review]

**Comment by armandgrillet** [comment]
`test-helm2` failing with
```
>>> helm dependency build ct_previous_revision874991636/stable/cert-manager-setup
Error building dependencies for previous revision of chart 'cert-manager-setup => (version: "0.2.5", path: "stable/cert-manager-setup")'
: Error waiting for process: exit status 1
>>> helm dependency build stable/cert-manager-setup
>>> git worktree remove ct_previous_revision874991636
Error installing charts: Error building dependencies for chart 'cert-manager-setup => (version: "0.2.5", path: "stable/cert-manager-setup")': Error waiting for process: exit status 1
------------------------------------------------------------------------------------------------------------------------
No chart changes detected.
```
Should I keep the version the same 🤔?

**Comment by hectorj2f** [comment]
@armandgrillet it does not make sense to me this `No chart changes detected.`

**Comment by jimmidyson** [comment]
@armandgrillet I expect helm2 to fail because afaik cert-manager chart is helm3 only now (CRD v1 API).

**Pull request merged**